doc/source/methods/Anchors.ipynb

```diff
@@ -64,7 +64,9 @@
     "\n",
```

Comment:
<p>What if <code>sampling_method="unknown"</code>?</p><p> </p><p>Typos:</p><p>simialar --&gt; similar</p><p>similiary --&gt; similarity</p><p>words most similar words --&gt; most similar words</p><p>expainer --&gt; explainer</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/432/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>added <code>unknown</code>  and fixed typos</p><p> </p>